ios/TeamHome/TeamHome/Models/Mock json/TagExample.json

```diff
@@ -0,0 +1,5 @@
+ {
```

Comment:
Property names in JSON files should be double quoted. This isn't properly formed JSON.